configure.d/1_bd_part_count.conf

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-. $(dirname $3)/conf_framework
+. $(dirname $3)/conf_framework.sh
```

Comment:
I think we should not update the copyright date in a files where the content doesn't change

Reply:
sorry, this comment is misplaced